src/main/scala/com/recipepuppy/query.scala

```diff
@@ -0,0 +1,28 @@
+package com
```

Comment:
Pedantic, but while this is supported, I don't think I've ever seen it.

Reply:
you mean explicitly nesting packages or what do you mean?

Reply:
Its easier to just package com.recipepuppy

Reply:
ah, I thought you could do that, but I ended up seeing this nested package stuff in cats in some places.